app/src/main/java/com/appdev/eateryblueandroid/ui/viewmodels/HomeViewModel.kt

```diff
@@ -18,7 +18,11 @@ class HomeViewModel(
     sealed class State {
         object Loading : State()
         data class Failure(val errorMsg: String) : State()
-        data class Data(val eateries: List<Eatery>, val sections: List<EaterySection>) : State()
+        data class Data(
+            val eateries: List<Eatery>,
+            val sections: List<EaterySection>,
+            val filters: List<String>
```

Comment:
How come we are using strings to determine the filters instead of EaterySections? I feel EaterySections would make the code in home\Main.kt much more concise (i.e. instead of if, if, if, if for every possible string filter--just iterate through all (Eatery) -> Boolean functions in filters list and check if eatery passes all of them).

Reply:
Oh, that's fascinating. I didn't realize that existed in the codebase. That's not bad, I think I'll do a quick rewrite for it.